auto_round/compressors/base.py

```diff
@@ -1790,7 +1805,7 @@ def quantize(self) -> tuple[torch.nn.Module, dict[str, Any]]:
         if len(all_blocks) > 1:
             pbar = tqdm(range(0, sum([len(i) for i in all_blocks]), self.nblocks))
         else:
-            pbar = None  # move the alg warning outside pbar
+            pbar = tqdm(range(0, len(all_blocks[0]), self.nblocks))  # move the alg warning outside pbar
```

Comment:
why changing this

Reply:
Flux has 2 blocks, main branch will close the pbar after the first block quantization is done.
So I put pbar both creation and close in quantize function.